package.json

```diff
@@ -18,12 +18,11 @@
     "start": "microbundle watch --no-compress --format modern,cjs --jsx React.createElement",
     "prepare": "run-s build",
     "prepack": "run-s build",
+    "lint": "eslint --ext .ts,.tsx ./",
+    "lint:fix": "yarn lint --fix",
     "test": "run-s test:unit test:lint test:build",
     "test:build": "run-s build",
-    "test:lint": "eslint --ext .ts,.tsx ./",
-    "test:unit": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
```

Comment:
I removed `react-scripts`, it was not used in any meaningful manner at all.
and we have no `jest` things anyways